docs/getting-started/installing/from-gui/index.en.md

```diff
@@ -0,0 +1,15 @@
+You can install Krux (both official or beta releases) onto your K210-based device using our
+official desktop application, [KruxInstaller](https://github.com/selfcustody/krux-installer),
+available for:
+
+> ⚠️  WARNING: Krux-Installer latest version is in it's alpha version. Maybe you can experience
```

Comment:
Is this warning as "alpha" still applicable?

Reply:
@qlrd still tagged as alpha.
Fixed all other typos and issues. Thanks!